metadata-models/src/main/pegasus/com/linkedin/identity/CorpGroupInfo.pdl

```diff
@@ -20,6 +20,7 @@ record CorpGroupInfo {
   @Searchable = {
     "fieldType": "TEXT_PARTIAL"
     "queryByDefault": true,
+    "enableAutocomplete": true,
```

Comment:
What this previously enabled and now it is removed? Curious about the reasoning for including this? Or did a customer ask for it?

Reply:
You're saying this was enabled previously and removed? I think it is natural to search for the display name of the group. I added this specifically because a customer was running a search and I'd suggested autocomplete for users/groups. I looked to write a test for this (in this pr in fact) and it didn't work because of missing annotations.